prototype-workingdir/Microsoft.Storage.Pickers/FileOpenPicker.cpp

```diff
@@ -0,0 +1,154 @@
+#include "pch.h"
+#include "FileOpenPicker.h"
+#include "FileOpenPicker.g.cpp"
+#include <windows.h>
+#include <shobjidl.h>
+#include <shobjidl_core.h>
+#include <winrt/Microsoft.UI.Interop.h>
+#include "PickerCommon.h"
+
+namespace winrt::Microsoft::Storage::Pickers::implementation
+{
+    FileOpenPicker::FileOpenPicker(winrt::Microsoft::UI::WindowId const& windowId)
+        : m_windowId(windowId)
+    {
+    }
+
+    winrt::Microsoft::Storage::Pickers::PickerViewMode FileOpenPicker::ViewMode()
+    {
+        return m_viewMode;
+    }
+    void FileOpenPicker::ViewMode(winrt::Microsoft::Storage::Pickers::PickerViewMode const& value)
+    {
+        m_viewMode = value;
+    }
+    hstring FileOpenPicker::SettingsIdentifier()
+    {
+        return m_settingsIdentifier;
+    }
+    void FileOpenPicker::SettingsIdentifier(hstring const& value)
+    {
+        m_settingsIdentifier = value;
+    }
+    winrt::Microsoft::Storage::Pickers::PickerLocationId FileOpenPicker::SuggestedStartLocation()
```

Comment:
As a WinUI 3 developer, I would appreciate it if we could provide any file path here. I understand that the goal is to keep it similar to the WinRT API, but the COM file picker doesn't have this limitation.   
Maybe by adding an additional `SuggestedStartLocationDirectory` that takes precedence if not set to null.